store/src/java/com/zimbra/cs/gal/GalSyncToken.java

```diff
@@ -208,8 +208,17 @@ public boolean isEmpty() {
     public void merge(GalSyncToken that) {
```

Comment:
Would benefit from a description explaining what merge is intended to achieve.

Reply:
I have added test case to explain.

Reply:
First token set after processing changes from mail_item table. 
Then token set after processing changes from tombstones.